### ISSUE TITLE
implement from for primitives -> Value-> JsonValue. Or use the serde_json::json! macro in indra code?  

### ISSUE THREAD

**Issue by jasongoodwin** [indradb/indradb#207]
If the dependency is on JsonValue, why not have a primitive From implementation that converts directly into JsonValue.

You guys have a lot of code that looks like this:

```
&models::JsonValue::new(serde_json::Value::Bool(true)
```

But you have this (maybe this came later?):

```
impl Into<indradb::JsonValue> for JsonValue {
    fn into(self) -> indradb::JsonValue {
        indradb::JsonValue::new(self.into())
    }
}
```

Which means you can simplify any statement of the first heuristic into this:

```
use serde_json::json;
&json!(true).into()
```

So I have a couple question, and bear with me as I'm still a bit of a rust noob, but I'm happy to help if you give me some advice here:

Is there a reason not to use the from implementation?
Also, could we not simplify the expressions further by implementing our own From for primitives into JsonValue?

```
            Value::Null => JsonValue::from(()),
            Value::Bool(b) => JsonValue::from(b),
            Value::Number(n) if n.is_u64() => JsonValue::from(n.as_u64().unwrap()),
            Value::Number(n) if n.is_i64() => JsonValue::from(n.as_i64().unwrap()),
            Value::Number(n) if n.is_f64() => JsonValue::from(n.as_f64().unwrap()),
...
```

Happy to help and just bear with me here :)

**Comment by ysimonson** [comment]
I suppose there's no reason for us to even expose `JsonValue` outside of lib. It's only there to implement some extra traits, so just taking in `serde_json::Value` everywhere likely doesn't carry overhead.

**Comment by ysimonson** [comment]
Fixed in #209. Since 3.0 hasn't been cut yet, this can go in as part of the release. Thanks for the suggestion, it made the API nicer!
